lib/spack/spack/cmd/ci.py

```diff
@@ -694,7 +699,7 @@ def ci_rebuild(args):
 
         # If this is a develop pipeline, check if the spec that we just built is
         # on the broken-specs list. If so, remove it.
-        if spack_is_develop_pipeline and "broken-specs-url" in ci_config:
+        if install_exit_code and spack_is_develop_pipeline and "broken-specs-url" in ci_config:
```

Comment:
@scottwittenburg I am not sure on this part. I think it is still correct to say that this package is now "not broken", but I blocked it